### PR TITLE
Add discourse

### PR DESCRIPTION
**Pull request by sroertgen**
https://github.com/hbz/lobid/issues/526

### CODE REVIEW

**Comment by sroertgen** [comment]
I think one ok review will be enough to merge this.

**Review by dr0i — APPROVED** [review]
thx!

**Review by acka47 — APPROVED** [review]
+1

File: src/components/bio.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ const Bio = () => {
   const social = data.site.siteMetadata?.social
 
   return (
-    <div className="bio">
+    <div  className="bio">
```
Comment (fsteeg) [inline src/components/bio.js]:
```suggestion
    <div className="bio">
```

**Review by fsteeg — APPROVED** [review]
Just a tiny blank typo, see inline suggestion.

**Pull request merged**